doc/TEMPLATES.md

```diff
@@ -274,6 +270,27 @@ html`<div>${bar}</div>`;
 // <div><span>♥1</span>…<span>♣A</span></div>
 ```
 
+#### DocumentFragment content binding
```

Comment:
Adds explainer for composing pre-instantiated DOM nodes via `DocumentFragment` instances.